src/GreenLightHealth.Client/wwwroot/js/site.js

```diff
@@ -2,3 +2,22 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+
+
+if (document.getElementById("accept")) {
+    document.getElementById("accept").addEventListener("click", agreeToHealthDec);
+    document.getElementById("decline").addEventListener("click", declineHealthDec);
+}
+
+//TODO figure out what needs to happen here, maybe just highlight for now
+function agreeToHealthDec() {
+    alert ("@User.Identity.Name agrees to health declaration");
```

Comment:
I don't think an alert will work for this. Once dismissed there won't be any persistent visual on screen.